### PR TITLE
NISRA content changes

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?

NISRA content changes.

### How to review

Ensure all changes  defined in the Trello card are complete.

### Checklist

- [x] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

#### Northern Ireland

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/nisra-content-changes/schemas/en/census_household_gb_nir.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/nisra-content-changes/schemas/en/census_individual_gb_nir.json)

### CODE REVIEW

**Comment by iwootten** [comment]
[[A] NISRA Final content changes (XS)](https://trello.com/c/fLCWkqkT/3977-a-nisra-final-content-changes-xs)

File: source/jsonnet/northern-ireland/individual/blocks/personal-details/date_of_birth.jsonnet [hunk 1]
```diff
@@ -4,13 +4,9 @@ local rules = import 'rules.libsonnet';
...
   title: title,
-  guidance: {
-    contents: [
-      {
-        description: 'For example, 31 3 1970',
-      },
-    ],
-  },
+  description: [
+    'For example, 31 3 1970',
```
Comment (iwootten) [inline source/jsonnet/northern-ireland/individual/blocks/personal-details/date_of_birth.jsonnet:8]: This date doesn't appear to match in the spreadsheet - year should be 1980

**Comment by iwootten** [comment]
This links are wonky on this

**Comment by iwootten** [comment]
> This links are wonky on this

File: source/jsonnet/northern-ireland/individual/blocks/personal-details/date_of_birth.jsonnet (hunk 1 above)
Comment (MebinAbraham) [reply source/jsonnet/northern-ireland/individual/blocks/personal-details/date_of_birth.jsonnet]: As discussed, didn't change cause the sheet was wrong. But checked with Donna and have updated to make it consistent.

**Pull request merged**